.github/actions/validate-endpoints/action.yml

```diff
@@ -87,6 +91,8 @@ runs:
 
     - name: Validate /app/info endpoint
       shell: bash
+      env:
+        VERSION_LABEL: ${{ inputs.version-label }}
```

Comment:
why not just use `${{ inputs.version-label }}` directly (like the other tests below)?